src/main/java/com/aws/greengrass/clientdevices/auth/iot/CertificateRegistry.java

```diff
@@ -100,6 +103,19 @@ public void updateCertificate(Certificate certificate) {
         runtimeConfiguration.putCertificate(certificateToCertificateV1DTO(certificate));
     }
 
+    /**
+     * Returns the PEMs that are stored for all the certificate ids that are being tracked.
+     */
+    public List<String> getAllCertificatePems() {
```

Comment:
I have concerns about this approach. Let's avoid loading everything in to memory all at once

Reply:
mmm what do you suggest ? - The reason I am doing this is because we need to update all the pems that we have stored. Do you want me to read one by one and execute the update?